generators/app/index.js

```diff
@@ -123,17 +163,27 @@ module.exports = class extends Generator {
         generator = generatorRepos[generatorIdx];
       }
 
-      const reqBranch = await octokit.repos.getBranch({
-        owner: this.options.ghOrg,
-        repo: generator.name,
-        branch: generator.default_branch,
-      });
+      // fetch the available branches to retrieve the latest commit SHA
```

Comment:
same here - error logging when no branch exists (e.g. for the middleware and the task repos)